week2/The system of the student's score management. v1.0.cpp

```diff
@@ -0,0 +1,160 @@
+#include <stdio.h>
+
+void readscore(int n, long student_number[], int score[]);
+void print_total_and_average(int n, int score[]);
+void rank_by_grade_in_descending_order(int n, long student_number[], int score[]);
+void print_score(int n, long student_number[], int score[]);
+void rank_by_student_number_in_ascending_order(int n, long student_number[], int score[]);
+int search_student_rank_and_grade_by_student_number(int n, long student_number[], int score[]);
+void count_the_percentage_of_the_each_category(int n, long student_number[], int score[]);
+
+int main()
+{
+	long student_number[30];
+	int score[30], n, _switch;
+	printf("1.Input record\n2.Caculate total and average score of course\n3.Sort in descending order by score\n");
+	printf("4.Sort in ascending order by number\n5.search by number\n6.Statistic analysis\n0.Exit\n\n");

```

Comment:
This is good but if you write in the way below, it will be much clearer. For usage information, I would suggest you always to do so.

```suggestion
        printf("1. Input record\n);
        printf("2. Caculate total and average score of course\n");
        printf("3. Sort in descending order by score\n");
	printf("4. Sort in ascending order by number\n");
	printf("5. Search by number\n");
	printf("6. Statistic analysis\n");
	printf("0. Exit\n\n");
```

Reply:
> > Function names are too long!
> > The lack of comment.
> > -.-
> 
> 1. Actually long function names won't be a problem because most of modern IDEs have name auto-completion support.
> 2. Nice to add some comments. For documentation comment, reference [here](https://github.com/torvalds/linux/blob/master/kernel/capability.c#L109-L115)

I know,thanks.